### DISCUSSION TITLE
Welcome to homeproxy Discussions!

### DISCUSSION THREAD

**Discussion by 1715173329** [immortalwrt/homeproxy#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chen19930618** [comment]
你好，联系不上之前的卖家了，你可以给我一个激活码吗，我重置了一下就这样了

**Comment by chen19930618** [comment]
你好，联系不上之前的卖家了，你可以给我一个激活码吗，我重置了一下就这样了
<img width="1976" height="1413" alt="屏幕截图" src="https://github.com/user-attachments/assets/2324c260-ba97-49ed-83b7-f85b93e494dc" />
